litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/AddProbe/index.tsx

```diff
@@ -0,0 +1,276 @@
+                {t('createWorkflow.tuneWorkflow.addProbe.labels.interval')}
+              </InputLabel>
+              <InputField
+                variant="primary"
+                width="50%"
+                id="interval"
+                type="number"
+                value={probeData.interval}
+                onChange={(e) =>
+                  setProbeData({ ...probeData, interval: e.target.value })
+                }
+              />
+            </div>
+            <div className={classes.formField}>
+              <InputLabel className={classes.formLabel} htmlFor="polling">
+                {t('createWorkflow.tuneWorkflow.addProbe.labels.polling')}
+              </InputLabel>
+              <InputField
+                variant="primary"
+                width="50%"
+                id="polling"
+                type="number"
+                value={probeData.pollingInterval}
+                onChange={(e) =>
+                  setProbeData({
+                    ...probeData,
+                    pollingInterval: e.target.value,
+                  })
+                }
+              />
+            </div>
+          </div>
+          <div className={classes.formField}>
+            <InputLabel className={classes.formLabel} htmlFor="interval-delay">
+              {t('createWorkflow.tuneWorkflow.addProbe.labels.intervalDelay')}
+            </InputLabel>
+            <InputField
+              variant="primary"
+              width="30%"
+              id="interval-delay"
+              type="number"
+              value={probeData.intervalDelaySeconds}
+              onChange={(e) =>
+                setProbeData({
+                  ...probeData,
+                  intervalDelaySeconds: e.target.value,
+                })
+              }
+            />
+          </div>
+          <hr className={classes.detailContainer} />
+          <div className={classes.subHeading}>
+            {t('createWorkflow.tuneWorkflow.addProbe.labels.probeDetails')}
+          </div>
+          <div className={classes.formField}>
+            <InputLabel className={classes.formLabel} htmlFor="url">
+              {t('createWorkflow.tuneWorkflow.addProbe.labels.url')}
+            </InputLabel>
+            <InputField
+              variant="primary"
+              width="50%"
+              id="url"
+              type="text"
+              value={probeData.url}
+              onChange={(e) =>
+                setProbeData({ ...probeData, url: e.target.value })
+              }
+            />
+          </div>
+          <div className={classes.formField}>
+            <InputLabel className={classes.formLabel} htmlFor="response">
+              {t('createWorkflow.tuneWorkflow.addProbe.labels.expectedRes')}
+            </InputLabel>
+            <InputField
+              variant="primary"
+              width="50%"
+              id="response"
+              type="number"
+              value={probeData.expectedResponseCode}
```

Comment:
The input schema of http probe has been changed, we don't this field now. Please refer: https://docs.litmuschaos.io/docs/litmus-probe/#httpprobe